### PR TITLE
fix: fixed linter/prettier setups and updated targets 

### LINKED ISSUE #1835: Fix linter setup for wallet

### PR DESCRIPTION
**Pull request by birtony**
Fixes #1835

Signed-off-by: Anton Biriukov <anton.biriukov@avast.com>

### CODE REVIEW

**Comment by rolsonquadras** [comment]
@birtony - This PR has conflicts

**Comment by birtony** [comment]
> @birtony - This PR has conflicts

@rolsonquadras resolved

File: .eslintrc.js [hunk 1]
```diff
@@ -55,7 +52,8 @@ module.exports = {
      * Disallow await inside of loops
      * https://eslint.org/docs/rules/no-await-in-loop
      */
-    'no-await-in-loop': 'error',
+    // TODO: fix related code and enable this rule
```
Comment (talwinder50) [inline .eslintrc.js:55]: Can we add issue number here with todo

File: .eslintrc.js [hunk 2]
```diff
@@ -87,10 +85,25 @@ module.exports = {
...
+
+    'mocha/max-top-level-suites': ['warn', { limit: 2 }],
+
+    /**
+     * Added temporary to disable error for defineProps in setup scripts in Vue components
+     */
+    'no-unused-vars': ['error', { varsIgnorePattern: 'props' }],
+
+    /**
+     * TODO: remove once corresponding issues are fixed
```
Comment (talwinder50) [inline .eslintrc.js:97]: same here

**Review by DRK3 — APPROVED** [review]

**Comment by DRK3** [comment]
Looks ok to me, but would recommend you have someone look at it who's more familiar with this project as well!

**Review by talwinder50 — APPROVED** [review]

**Pull request merged**